src/requirements.txt

```diff
@@ -1,26 +1,26 @@
 -i https://pypi.python.org/simple
-asgiref==3.6.0 ; python_version >= '3.7'
-boto3==1.26.69
-botocore==1.29.69 ; python_version >= '3.7'
-cachetools==5.3.0
-certifi==2022.12.7 ; python_version >= '3.6'
+asgiref==3.7.2 ; python_version >= '3.7'
+boto3==1.26.144
+botocore==1.29.144 ; python_version >= '3.7'
+cachetools==5.3.1
+certifi==2023.5.7 ; python_version >= '3.6'
```

Comment:
🧵 Major version bump

Reply:
- https://pypi.org/project/certifi/#history
- https://pypi.org/project/certifi/2023.5.7/
- https://github.com/certifi/python-certifi/releases/tag/2023.05.07

Not finding release notes or a changelog 👀

Reply:
Not semver, date releases for certificate store, should not be an issue, or we'll see it in smoke testing.